src/PscIde/Server.purs

```diff
@@ -46,7 +47,8 @@ type PscIdeServerArgs = {
 
 defaultServerArgs :: PscIdeServerArgs
 defaultServerArgs = {
-  exe: "psc-ide-server",
+  exe: "purs",
+  combinedExe: true,
```

Comment:
Maybe just add a TODO note here, so that we can deprecate and remove this at some point.